src/Stryker.Core/Stryker.Core/MutationTest/MutationTestProcess.cs

```diff
@@ -208,5 +214,18 @@ public StrykerRunResult Test(StrykerOptions options)
 
             return new StrykerRunResult(options, _input.ProjectInfo.ProjectContents.GetMutationScore());
         }
+
+        public TestCoverageInfos GetCoverage()
+        {
+            var (compat_noAppDomain, compat_noPipe) = _input.ProjectInfo.ProjectUnderTestAnalyzerResult.CompatibilityModes;
```

Comment:
The variable names `compat_noAppDomain` and `compat_noPipe` are not very clear to me.

Reply:
ok